xml/System.Diagnostics/DistributedContextPropagator.xml

```diff
@@ -54,8 +54,8 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>returns the default propagator object which Current property will be initialized with.</summary>
-        <returns>To be added.</returns>
+        <summary>Returns the default propagator object which Current property will be initialized with.</summary>
```

Comment:
Added the full cref.
Instead of `which`, should it be `that the`?:


```suggestion
        <summary>Returns the default propagator object which <see cref="P:System.Diagnostics.DistributedContextPropagator.Current"/> property will be initialized with.</summary>
```